src/shapes/polygon.cpp

```diff
@@ -0,0 +1,83 @@
+#include "shapes/polygon.hpp"
+#include "shapes/star.hpp"
+#include "shapes/straight_vertex.hpp"
+#include <cmath>
+
+using namespace rive;
+
+Polygon::Polygon() {}
```

Comment:
Total nitpick, but did it complain if you didn't have the constructors defined? Might be nice to not have the empty ones laying around, but don't let that prevent you from merging this...

Reply:
Yeah I wanted to leave them out but it wouldn't compile without them